Source/Core/Core/TitleDatabase.cpp

```diff
@@ -91,8 +94,26 @@ TitleDatabase::TitleDatabase()
 
 TitleDatabase::~TitleDatabase() = default;
 
+const std::string& TitleDatabase::GetDeveloper(const std::string& gametdb_id) const
+
```

Comment:
Could you remove the empty line here and the empty line you added to the function below? It seems like lint isn't complaining about it, but we never put an empty line here.

Reply:
Yes.